x-pack/platform/plugins/shared/lens/public/visualizations/xy/visualization.tsx

```diff
@@ -44,7 +44,9 @@ import {
   getColorMappingDefaults,
 } from '../../utils';
 import { getSuggestions } from './xy_suggestions';
-import { XyToolbar, updateLayer } from './xy_config_panel';
+import { XyToolbar } from './toolbar';
+import { XyFlyoutToolbar } from './toolbar/flyout_toolbar'; // TODO: Update import
+import { updateLayer } from './toolbar'; // TODO: Update this import
```

Comment:
If these todos (and other I've seen in the PR) are considered for a secondary step, please remove them from here, move them into a GH issue. If instead this is something you are considering to address in this PR or was just forgot, please remove them.

It is alsostrange that the `XyToolbar` and the `updateLayer` imports in the same import